composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name":        "roave/don-t",
+    "description": "A small set of traits that can be used to prevent misuse of your objects",
+    "type":        "library",
+    "license":     "MIT",
+    "homepage":    "https://github.com/Roave/Dont",
+    "keywords":    [
+        "defensive programming",
+        "strictness"
+    ],
+    "authors": [
+        {
+            "name":     "Marco Pivetta",
+            "email":    "ocramius@gmail.com",
+            "homepage": "http://ocramius.github.io/"
+        }
+    ],
+    "require": {
+        "php": "7.*"
```

Comment:
We can probably push to 7.1. This package is initially to be deployed as part of `ocramius/proxy-manager` for a version that only supports PHP 7.1+